src/irmin-git/irmin_git.ml

```diff
@@ -307,7 +307,8 @@ struct
               let v k = (l, k) in
               match x with
               | `Node n -> to_git `Dir (v n)
-              | `Contents (c, perm) -> to_git (perm :> G.Value.Tree.perm) (v c))
+              | `Contents (c, perm) -> to_git (perm :> G.Value.Tree.perm) (v c)
+              )
```

Comment:
This closing paren looks like it should be in the same column as the opening one. I find this wrapping quite confusing to look at.